pandas/core/groupby/indexing.py

```diff
@@ -112,7 +112,9 @@ def _positional_selector(self) -> GroupByPositionalSelector:
         4  b  5
         """
         if TYPE_CHECKING:
-            groupby_self = cast(groupby.GroupBy, self)
+            groupby_self = cast(
+                groupby.GroupBy, self  # pylint: disable=used-before-assignment
+            )
```

Comment:
same comment as below, can you put `disable-next` before this line so we avoid splitting it?

Reply:
Got it! Thanks!